src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/tasks/CleanupEnvelopesDlqTask.java

```diff
@@ -100,14 +102,19 @@ private void logMessage(IMessage msg) {
     }
 
     private boolean canBeCompleted(IMessage message) {
-        Instant createdTime = message.getEnqueuedTimeUtc();
         Instant cutoff = Instant.now().minus(this.ttl);
-        boolean canBeCompleted = createdTime.isBefore(cutoff);
+        boolean canBeCompleted;
+
+        if (message.getProperties() != null && isNotEmpty(message.getProperties().get("deadLetteredAt"))) {
+            canBeCompleted = Instant.parse(message.getProperties().get("deadLetteredAt")).isBefore(cutoff);
+        } else {
+            canBeCompleted = message.getEnqueuedTimeUtc().isBefore(cutoff);
```

Comment:
As discussed we can remove this condition and instead just log the envelope id so that we are aware of any message getting dead lettered without `deadLetteredAt` property.

Reply:
sure